includes/class-wc-install.php

```diff
@@ -101,6 +101,7 @@ class WC_Install {
 			'wc_update_330_product_stock_status',
 			'wc_update_330_set_default_product_cat',
 			'wc_update_330_db_version',
+			'wc_update_330_clear_transients',
```

Comment:
Most trivial of all nitpicks and I'm sorry. :) The db_version is always the last thing in the array. That way if something crashes during the upgrade process the DB version will still be the old version and you won't be stuck with a failed upgrade. Once that's changed, this is good to merge.